### PR TITLE
6195 gps icon

### PR DESCRIPTION
**Pull request by gdicristofaro**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/datamodel/ExtractedContent.java [hunk 1]
```diff
@@ -154,7 +154,8 @@ static String getIconFilePath(int typeID) {
             return filePath + "encrypted-file.png"; //NON-NLS
         } else if (typeID == BlackboardArtifact.ARTIFACT_TYPE.TSK_EXT_MISMATCH_DETECTED.getTypeID()) {
             return filePath + "mismatch-16.png"; //NON-NLS
-        } else if (typeID == BlackboardArtifact.ARTIFACT_TYPE.TSK_GPS_ROUTE.getTypeID()) {
+        } else if (typeID == BlackboardArtifact.ARTIFACT_TYPE.TSK_GPS_ROUTE.getTypeID()
+                || typeID == BlackboardArtifact.ARTIFACT_TYPE.TSK_GPS_ROUTE.getTypeID()) {
```
Comment (ethteck) [inline Core/src/org/sleuthkit/autopsy/datamodel/ExtractedContent.java]: It looks like this conditional is checking for the same thing. Maybe I'm missing something though

Comment (gdicristofaro) [reply Core/src/org/sleuthkit/autopsy/datamodel/ExtractedContent.java]: you're right.  sorry about that.

Comment (ethteck) [reply Core/src/org/sleuthkit/autopsy/datamodel/ExtractedContent.java]: np!

**Comment by ethteck** [comment]
LGTM! I don't have write permissions on the repo, so feel free to merge.

**Pull request merged**